lib/appearance.rb

```diff
@@ -93,7 +93,10 @@ def element_frequency(element)
     #
     # Returns the String
     def inner_word
-      html = Nokogiri::HTML(open(url))
+      html_content = Rails.cache.fetch("html", :expires_in => 1.day) do
```

Comment:
INFO: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)